pkgs/servers/etebase/default.nix

```diff
@@ -9,13 +8,22 @@
 , redis
 , typing-extensions
 , withLdap ? true
+, python3
 , python-ldap
 , withPostgres ? true
 , psycopg2
 , nix-update-script
+, nixosTests
 }:
 
-buildPythonPackage rec {
+let
+  python = python3.override {
```

Comment:
We should also use below python to make sure we are not mixing versions of Python or run into conflicts.

Reply:
WDYM? We are using this python for the default package. Users could still override for custom packages as needed?

Reply:
We are not using it for the other dependencies like django, etc.